openedx_learning/apps/authoring/collections/api.py

```diff
@@ -22,15 +22,15 @@
 
 def create_collection(
     learning_package_id: int,
-    name: str,
+    title: str,
     description: str = "",
 ) -> Collection:
     """
     Create a new Collection
     """
     collection = Collection.objects.create(
         learning_package_id=learning_package_id,
-        name=name,
+        title=title,
         description=description,
     )
```

Comment:
```suggestion
        description=description,
        created_by=created_by,
    )
```

Likewise we need to add the `created_by` here